docs/recon_configurations/README.md

```diff
@@ -108,6 +109,15 @@ class Table:
 {
   "source_name": "&lt;SOURCE_NAME&gt",
   "target_name": "&lt;TARGET_NAME&gt",
+  "aggregates": [{
```

Comment:
Move this to different table like how we have explained `jdbc_reader_options` etc.

Reply:
Done